modules/tensor_mechanics/include/materials/ComputeElasticityTensorBase.h

```diff
@@ -18,6 +18,8 @@ class ComputeElasticityTensorBase : public DerivativeMaterialInterface<Material>
 public:
   ComputeElasticityTensorBase(const InputParameters & parameters);
 
+  virtual bool isGuaranteedIsotropic() const { return false; }
```

Comment:
Method name speaks for itself, but probably a doco comment would be good nevertheless